crates/oxc_query/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+#![feature(let_chains)]
+#![allow(clippy::redundant_pub_crate)]
+mod adapter;
+mod edges;
+mod entrypoints;
+mod properties;
+mod util;
+mod vertex;
+
+pub use adapter::{schema, Adapter};
+pub use vertex::Vertex;
+
+// TODO: Uncomment on next release of trustfall
```

Comment:
this comment is already out of place...

Reply:
I'll just remove this for now and pr it back in the next trustfall release

Reply:
[d463ba4](https://github.com/web-infra-dev/oxc/pull/599/commits/d463ba4a390df0206564c562e55bdf3ceb826b14)